portfolio/src/main/java/com/google/sps/servlets/USPopGrowthDataServlet.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/pop-data")
+public class USPopGrowthDataServlet extends HttpServlet {
+    private static final String JSON_TYPE = "application/json";
+    private LinkedHashMap<Integer, Double> populationGrowth = new LinkedHashMap<Integer, Double>();
```

Comment:
why use a LinkedHashMap over a HashMap?

Reply:
A LinkedHashMap keeps the order that the data was inserted, HashMap doesn't.

Reply:
Gotcha. Since this data is inherently ordered, a simple List might make the most sense. This is fine with me though.